UI/data/themes/Rachni.qss

```diff
@@ -0,0 +1,1032 @@
+/***************************************************************************/
+/*   Copyright (C) 2017-2018 by Joel Bethke <joel.bethke@gmail.com>        */
+/*                                                                         */
+/*                                                                         */
+/*   This program is free software: you can redistribute it and/or modify  */
+/*   it under the terms of the GNU General Public License as published by  */
+/*   the Free Software Foundation, either version 2 of the License, or     */
+/*   (at your option) any later version.                                   */
```

Comment:
Compare to the license at the top of this random libobs file I grabbed: https://github.com/jp9000/obs-studio/blob/master/libobs/obs-audio.c#L1-L16